ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Stream.scala

```diff
@@ -92,6 +91,9 @@ private[h2] class H2Stream[F[_]: Concurrent](
         }
         .compile
         .drain
+        .onError { case _ =>
+          rstStream(H2Error.InternalError)
```

Comment:
If you are writing a stream you need to close the stream in some capacity. This says the stream is broken.